### PR TITLE
adds zeno

### PR DESCRIPTION
**Pull request by zenorocha**

### CODE REVIEW

**Comment by kevva** [comment]
Cool, welcome to the list!

File: readme.md [hunk 1]
```diff
@@ -83,6 +83,7 @@
 - [Vladimir Agafonkin](https://github.com/mourner/ama) - Engineer at [Mapbox](https://www.mapbox.com/), creator of [Leaflet](http://leafletjs.com/) and a bunch of [other open source libraries](https://github.com/mourner/projects). Building the future of interactive maps.
 - [Claudio Procida](https://github.com/claudiopro/ama) - Full stack web developer, mentor, evangelist and speaker. Passionate about painting and photography.
 - [Jordan Scales](https://github.com/jdan/ama) - Engineer at [Khan Academy](https://khanacademy.org), creator of the [tota11y](http://khan.github.io/tota11y) accessibility toolkit and [isomer](http://jdan.github.io/isomer).
+- [Zeno Rocha](https://github.com/zenorocha/ama) - Developer Advocate at [Liferay](http://www.liferay.com/)
```
Comment (shinnn) [inline readme.md:86]: Needs the period. https://github.com/sindresorhus/amas/blob/master/contributing.md

``` diff
- [Liferay](http://www.liferay.com/)
+ [Liferay](http://www.liferay.com/).
```

**Comment by zenorocha** [comment]
;D

File: readme.md (hunk 1 above)
Comment (kevva) [inline readme.md:86]: Yeah, I noticed right after I merged, but I edited the commit.

**Comment by sindresorhus** [comment]
Welcome Zeno! :)

Would be cool if you could ask some questions on the other AMAs in the list (not my AMA). I think AMAs are a great opportunity to learn from each other. Asking a question is also a good way to encourage receiving one ;)

For inspiration of what to ask, you could take a look at the questions in some other AMAs, like:
https://github.com/passy/ama/issues?q=
https://github.com/sebmck/ama/issues?q=
https://github.com/wesbos/ama/issues?q=

**Pull request merged**